files/jira.rb

```diff
@@ -5,10 +5,16 @@
 class Jira < BaseHandler
 
   def build_labels
-    [ "SENSU_#{@event['client']['name']}",
+    user_labels = [*(@event['check']['tags'] || [])]
```

Comment:
Are we guaranteed these will be lists and not hashes? We can double splat like I did before but that doesn't work for Ruby 1.8 (looks like you can only have a single splat ... idk).

If we're guaranteed these will be lists I can make this change (my change fixed a bug for 1.8 before where we splatted nil and would append nil to the labels ...).